test/read-file-sync.test.js

```diff
@@ -43,7 +43,7 @@ describe('+ readFileSync()', function () {
         jf.readFileSync(file)
       }, function (err) {
         assert(err instanceof Error)
-        assert(err.message.match(file))
+        assert(err.message.match(/somefile\.json/))
```

Comment:
Have you tried echoing out `err.message` and `file` to see why it doesn't match?